.gitlint

```diff
@@ -0,0 +1,7 @@
+[general]
+ignore=body-is-missing
+
+# TODO(robinlinden): Better way of documenting and setting this up.
+# Each commit must start with the main area it affects.
```

Comment:
If only there was a way to ask git for all commits that touch a file/directory

Reply:
We're doing it this way because of the parallel implementations of things, along with multiple tokenizers and parsers etc. Instead of "handle comments in css tokenization in the old css library" we write "css: handle comment tokenization". Makes disambiguating between the things shorter and normalized.